pkg/network/subnet_test.go

```diff
@@ -0,0 +1,34 @@
+package network
+
+import (
+	"net"
+	"reflect"
+	"testing"
+)
+
+func TestNextSubnet(t *testing.T) {
+	type args struct {
+		subnet *net.IPNet
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *net.IPNet
+		wantErr bool
+	}{
+		{"class c", args{subnet: parseCIDR("192.168.0.0/16")}, parseCIDR("192.169.0.0/16"), false},
```

Comment:
FYI, this is a Class B, the next one's a Class C - and even then, those are pretty outdated, since they refer back to the classful addressing days before CIDR was a thing.

Reply:
updated, the tests are still valid